### PR TITLE
Feature/refactor link component

### PR DESCRIPTION
**Pull request by hdfeddanjackson**

### CODE REVIEW

File: components/atoms/link/Link.js [hunk 1]
```diff
@@ -1,28 +1,36 @@
...
+	const linkClass = classNames('a-link', className, {
 		'a-link--primary': theme === 'primary',
-		'a-link--secondary': theme === 'secondary',
-	});
+		'a-link--secondary': theme === 'secondary'
+	})
 
 	return(
-		<a href={href} className={linkClass} rel={rel} role={role} target={target}>{children}</a>
+		<a {...{ href, rel, role, target }} className={linkClass}>{children}</a>
```
Comment (hdfedscottquested) [inline components/atoms/link/Link.js:23]: The spread operator...Nice!

**Review by hdfedscottquested — APPROVED** [review]
Approved

File: components/atoms/link/Link.js (hunk 1 above)
Comment (hdfeddanjackson) [reply components/atoms/link/Link.js:23]: Thanks, helps to tidy-up the code. If you have a lot of props in a component that have the same name and values, like:

```javascript
<a href={href} rel={rel}>{children}</a>
```

It's easier to just use the spread operator:

```javascript
<a href={...{ href, rel }}>{children}</a>
```

**Review by hd-darrenwhatford — APPROVED** [review]
Makes sense and code look good to me now.

Approved

**Pull request merged**